crates/cheatcodes/src/evm.rs

```diff
@@ -3,14 +3,15 @@
 use crate::{Cheatcode, Cheatcodes, CheatsCtxt, Result, Vm::*};
 use alloy_primitives::{Address, Bytes, U256};
 use alloy_sol_types::SolValue;
+use ethers_core::utils::GenesisAccount;
 use ethers_signers::Signer;
 use foundry_evm_core::backend::DatabaseExt;
 use foundry_utils::types::ToAlloy;
 use revm::{
     primitives::{Account, Bytecode, SpecId, KECCAK_EMPTY},
     EVMData,
 };
-use std::collections::HashMap;
+use std::{collections::HashMap, fs::File};
```

Comment:
this should use foundry's fs bindings

Reply:
done in https://github.com/foundry-rs/foundry/pull/6207/commits/1292c7b9856b1391d06a456398ab2417f2831134